ENIGMAsystem/SHELL/Graphics_Systems/OpenGLES/GLESscreen.cpp

```diff
@@ -121,34 +117,34 @@ int screen_redraw()
           if (view_yview[vc]>room_height-view_hview[vc]) view_yview[vc]=room_height-view_hview[vc];
         }
       }
-      
+
       glViewport((int)view_xport[vc],(int)view_yport[vc],(int)view_wport[vc],(int)view_hport[vc]);
       glLoadIdentity();
       glScalef(1,-1,1);
 #if ENIGMA_WS_IPHONE !=0 || ENIGMA_WS_ANDROID !=0
 		glOrthof((int)view_xview[vc],(int)view_wview[vc]+(int)view_xview[vc],(int)view_yview[vc],(int)view_hview[vc]+(int)view_yview[vc],0,1); //OPENGLES put f back in
 #else
       glOrtho((int)view_xview[vc],(int)view_wview[vc]+(int)view_xview[vc],(int)view_yview[vc],(int)view_hview[vc]+(int)view_yview[vc],0,1); //OPENGLES put f back in
-#endif    
+#endif
       if (background_showcolor)
       {
          int clearcolor=((int)background_color)&0xFFFFFF;
-         glClearColor(__GETR(clearcolor)/255.0,__GETG(clearcolor)/255.0,__GETB(clearcolor)/255.0, 1);
+         glClearColor(COL_GET_R(clearcolor)/255.0,COL_GET_G(clearcolor)/255.0,COL_GET_B(clearcolor)/255.0, 1);
          glClear(GL_COLOR_BUFFER_BIT);
       }
       draw_back();
-        
+
       for (enigma::instance_event_iterator = event_draw->next; enigma::instance_event_iterator != NULL; enigma::instance_event_iterator = enigma::instance_event_iterator->next)
         enigma::instance_event_iterator->inst->myevent_draw();
     }
-    
+
     return 0;
 }
 
 
-#undef __GETR
-#undef __GETG
-#undef __GETB
+#undef COL_GET_R
```

Comment:
This something we want individual sources to be doing?

Reply:
Not really, and I also did not notice this either. I don't even know if it's necessary to be honest, but this system doesn't work either.

Reply:
I followed the git blame back to: 29f857ec1eacd84e846d2966707c6be395e7f8c3
More specifically: https://github.com/enigma-dev/enigma-dev/commit/29f857ec1eacd84e846d2966707c6be395e7f8c3#diff-2064c9a99041f4ea987c51f5c17554e6R131

So, it looks like it was always that way, and he didn't need to do that.